lib/workspace.js

```diff
@@ -528,7 +542,7 @@ module.exports = {
     const archive = archiver('zip');
     const s3Name = `workspace-${workspace_id}-${workspace_version}`;
     await async.eachLimit(
-      workspace_graded_files,
+      workspace_graded_files, // wildcards are not supported in S3, as it is not in use in production
```

Comment:
Ideally I'd like to merge https://github.com/PrairieLearn/PrairieLearn/pull/6983 first so that we don't need to have this at all.

Reply:
I have removed the comment and the related S3 code in #6947 so a conflict resolution in either PR should be easier, depending on which one is merged first.